jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/SpringBootExplodedProcessor.java

```diff
@@ -41,20 +41,20 @@
 public class SpringBootExplodedProcessor implements JarProcessor {
 
   @Nullable private final Path jarPath;
-  @Nullable private final Path tempDirectoryPath;
+  @Nullable private final Path targetDirectoryPath;
```

Comment:
Remove `@Nullable` for `jarPath` and `targetDirectoryPath`, and double-check that indeed they cannot be null. Do you remember why they started with `@Nullable`?

Reply:
This is a great catch, thank you! Yes `targetDirectoryPath` and `jarPath` cannot be null.`@Nullable` was added when I was refactoring `"JarModeProcessor"` into `StandardExplodedProcessor`, `SpringBootExplodedProcessor`, etc. I was initially using setters to set the `tempDirectoryPath` and `jarPath` since not all variations of `createLayers` and `computeEntrypoint` use **both** these parameters. The design changed to allow these values be passed in at a constructor level but I forgot to remove the `@Nullable`.